biz.aQute.bndlib/src/aQute/bnd/osgi/Constants.java

```diff
@@ -98,6 +98,12 @@
 	String							DEPLOYREPO									= "-deployrepo";
 	String							DIGESTS										= "-digests";
 	String							DSANNOTATIONS								= "-dsannotations";
+	String							DSANNOTATIONS_OPTIONS						= "-dsannotations-options";
+	// -dsannotations-options values
+	String							INHERIT										= "inherit";
```

Comment:
The option constant name should be prefixes with the instruction to which they are associated. E.g. DSANNOTATIONS_OPTIONS_INHERIT.